amaranth/sim/_pyrtl.py

```diff
@@ -345,7 +355,29 @@ def gen(arg):
         return gen
 
 
+def value_to_string(value):
```

Comment:
Maybe something like this?

```suggestion
def value_to_string(value):
    """Unpack a Verilog-like (but LSB-first) string of unknown width from an integer."""
```